File: Challenges/Java/FareEstimator.java
import java.util.stream.IntStream;

final class FareEstimator {

	double[] fareEstimator(int ride_time, int ride_distance, double[] cost_per_minute, double[] cost_per_mile) {
    
	    return IntStream.range(0, cost_per_mile.length)
	    .mapToDouble(i -> cost_per_minute[i] * ride_time + cost_per_mile[i] * ride_distance)
	    .toArray();
	}
}

File: Challenges/Javascript/fareEstimator.js
function fareEstimator(ride_time, ride_distance, cost_per_minute, cost_per_mile) {
    let out = [];
    
    for(let i = 0; i < cost_per_mile.length; i++)
        out[i] = cost_per_minute[i] * ride_time + cost_per_mile[i] * ride_distance;
    
    return out;
}